schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/MDTDeliveryProtocol.avdl

```diff
@@ -108,55 +20,10 @@ protocol MDTDelivery {
               approved by the originating GMCs
     */
     record InterpretationData {
-        TieringResult tieringResult;
-        array<InterpretedGenomeRD> interpretationResults;
+        InterpretationRequestRD InterpretationMetaData;
```

Comment:
Please respect camel case for attribute names, this messes up with Java de/serialisation.